src/Arcus.WebApi.Unit/Security/Configuration/SecretProviderConfigurationBuilderExtensionsTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.Extensions.Configuration;
+using Xunit;
+using Arcus.WebApi.Security.Configuration;
+using System.Collections.Generic;
+
+namespace Arcus.WebApi.Unit.Security.Configuration
+{
+    public class SecretProviderConfigurationBuilderExtensionsTests
+    {
+        [Fact]
+        public void AddKeyVault_Accesses_SecretProvider_For_Secret_Values_From_Configuration_Tokens()
```

Comment:
Please align to the current naming convention that we have

Reply:
Yes, but not sure how to use this in this case?

Reply:
Ah, fixed this.